Automattic/simplenote-macos
# Add PRs list generator lane

This PR adds the get_pullrequests_list lane to Fastlane.
The command generates a list of all the PRs that have been merged since the provided tag.

## To Test:
1. Copy `fastlane/env.example` to `~/.simplenotemacos-env.default` and set access rights to 600.
2. Setup the env file:
2.a In GitHub, open the Personal Access Token page.
2.b Click Generate New Token.
2.c Grant repo permission to the token.
2.d Click Generate Token.
2.e Copy the Token key and paste it into the `GHHELPER_ACCESS` field of `~/.simplenotemacos-env.default`.
3. Run bundle install.
4. From the project root folder, run `bundle exec fastlane get_pullrequests_list` start_tag:1.4.1 (you can use any valid tag as start_tag. 1.4.1 is just the last one).
5. Verify that the `simplenotemacos_prs_list.txt` file has been generated in your home folder with the list of the merged PRs.
